### ISSUE TITLE
Update Nuget Package

### ISSUE THREAD

**Issue by chuck-flowers** [ironyproject/irony#6]
I'm interested in trying out Irony for a project idea I've had but it appears the 'Irony' nuget package only targets .NET Framework 4.0. I'm wanting to use it for a .NET Standard/Core project. From the issue tracker here it seems like you've added .NET Standard support but the nuget package doesn't seem to reflect that. 

Does the package need to be updated or am I missing something?

**Comment by yallie** [comment]
Hi Chuck,

you're right, the Nuget package is out of date.
I don't have access to the package, perhaps Roman does.
For now, you can just build Irony code from sources.

**Comment by chuck-flowers** [comment]
Alright thanks.

**Comment by igitur** [comment]
On that subject, is Roman involved with this fork?

**Comment by yallie** [comment]
Roman is the administrator of the IronyProject organization.
However I'm not sure if he has time to actively contribute to Irony.

**Comment by igitur** [comment]
Ok, let me rephrase, does this fork have his 'blessing'? If so, let's get @daxnet involved with this fork rather.

**Comment by igitur** [comment]
And what's the chances of getting control of the NuGet package?  cc @rivantsov ?

**Comment by rivantsov** [comment]
Hi guys
this is Roman. I just granted control over nuget packages to Alexey (yallie), so from now on he is in charge. Whatever he does, he 'has my blessing'.
 Sorry, no time to be involved, too much other stuff going on. There is a possibility I might come back, I am planning GraphQL server implementation (on top of VITA (https://github.com/rivantsov/vita), and it turns out search (predicate language) is not part of 'standard', so it's up to implementers to figure it out

**Comment by igitur** [comment]
Thanks, Roman.

**Comment by igitur** [comment]
@yallie Would you mind downloading https://ci.appveyor.com/api/buildjobs/152pwkrfd38p08xq/artifacts/Irony%2Fbin%2FRelease%2FIrony.1.0.9.nupkg , adjusting its internal version number and pushing it to NuGet as a beta version. I suggest using version number `1.0.0-beta1` .

**Comment by yallie** [comment]
Looks like the [package](https://www.nuget.org/packages/Irony) currently includes the GrammarExplorer tool.
Perhaps it should better be published as a separate nuget, say, Irony.GrammarExplorer.
I guess I'm publishing the package as is for now, just Irony.dll.

@igitur @wasabii Any thoughts?

**Comment by igitur** [comment]
Definitely agree that the tools should be published separately. I'll add the CI .nupkg build for that later.

**Comment by musukvl** [comment]
I don't see Irony.Interpreter beta in nuget latest I see is 0.9.1.
Where can I get Irony.Interpreter  for my .net core project?

**Comment by yallie** [comment]
@musukvl, just published this:
https://www.nuget.org/packages/Irony.Interpreter/1.0.0-beta1

**Comment by wasabii** [comment]
Why hello there. Looks like everybody has everything handled? Heh.

Yes. The tools should be split out. The original package was a quick effort on my part. I didn't even get it upstream, I don't think. Mercurial. Ugh.

**Comment by yallie** [comment]
Hello @wasabii,

>The tools should be split out. 

Yes, that's on the todo-list.
